valkey.conf

```diff
@@ -150,10 +150,10 @@ tcp-backlog 511
 # Unix socket.
 #
 # Specify the path for the Unix socket that will be used to listen for
-# incoming connections. There is no default, so Redis will not listen
+# incoming connections. There is no default, so the server will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /run/redis.sock
+# unixsocket /run/server.sock
```

Comment:
should we change this to valkey.sock instead? I wonder if a generic name like this would increase the collision risk in the file system namespace.